filer/templates/admin/filer/widgets/admin_file.html

```diff
@@ -14,7 +14,7 @@
         </span>
     </div>
 
-    <div class="js-filer-dropzone filer-dropzone{% if object %} js-object-attached{% endif %}"
+    <div class="js-filer-dropzone filer-dropzone{% if object %} js-object-attached image-preview-container{% endif %} "
```

Comment:
```suggestion
    <div class="js-filer-dropzone filer-dropzone{% if object %} js-object-attached{% endif %} "
```
I think the actual bug is in line 29, where it should say `detail="thumbnail". Would you mind verifying that?